### PR TITLE
Feat: Text utilities

### PR DESCRIPTION
**Pull request by Muzammil-khan710**
This is a small PR .
I have added text utilities

### CODE REVIEW

File: docs/components/Text-utilities/text-util.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+.heading-five {
+    font-size: 1.5rem;
+}
+
+.heading-six {
+    font-size: 1.2rem;
+}
+
+.small-text {
```
Comment (developer-anand) [inline docs/components/Text-utilities/text-util.css:26]: you can use rem for fontsize too.

**Review by developer-anand — APPROVED** [review]
Can be merged. Good naming conventions :)

File: docs/components/Text-utilities/text-util.css (hunk 1 above)
Comment (Muzammil-khan710) [reply docs/components/Text-utilities/text-util.css:26]: It was 0.9 rem so i switched to px to make it clean

**Review by Shubhamkr2610 — APPROVED** [review]
Looks fine , code is clean and readable . Easy to understand . you can merge it

**Pull request closed without merging**